WooCommerce/Classes/ViewRelated/CardPresentPayments/CardPresentPaymentsModalViewController.swift

```diff
@@ -281,7 +281,9 @@ private extension CardPresentPaymentsModalViewController {
     func configureAuxiliaryButton() {
 
         guard shouldShowAuxiliaryButton() else {
-            auxiliaryButton.isHidden = true
+            // Only for testing, switch back to true before merging
+            //auxiliaryButton.isHidden = true
+            auxiliaryButton.isHidden = false
```

Comment:
This is for convenience when testing the Case 2, has to be removed before merging.